client/state/action-types.js

```diff
@@ -25,6 +25,8 @@ export const ANALYTICS_MULTI_TRACK = 'ANALYTICS_MULTI_TRACK';
 export const ANALYTICS_PAGE_VIEW_RECORD = 'ANALYTICS_PAGE_VIEW_RECORD';
 export const ANALYTICS_STAT_BUMP = 'ANALYTICS_STAT_BUMP';
 export const AUTOMATED_TRANSFER_STATUS_SET = 'AUTOMATED_TRANSFER_STATUS_SET';
+export const AUTOMATED_TRANSFER_ELIGIBILITY_REQUEST = 'AUTOMATED_TRANSFER_ELIGIBILITY_REQUEST';
+export const AUTOMATED_TRANSFER_ELIGIBILITY_UPDATE = 'AUTOMATED_TRANSFER_ELIGIBILITY_UPDATE';
```

Comment:
Super minor: wrong alphabetical order (those two constants should come before `AUTOMATED_TRANSFER_STATUS_SET`).